### ISSUE TITLE
Multiply y-axis

### ISSUE THREAD

**Issue by rolyanos** [danielgindi/Charts#2165]
@PhilJay, hello, I have a question about multiply y-axis for line chart, is that support for current version?

Thanks a lot.

**Comment by liuxuan30** [comment]
Not yet. just left and right now.

**Comment by rolyanos** [comment]
So sad, is it planning in the future?

**Comment by liuxuan30** [comment]
I don' think so. Since more than 2 y axis seems not that popular. Some  other project may have, or you can definitely develop based on this library.

**Comment by rolyanos** [comment]
I do not agree, when we have more than 3-5 different values(lines, ex: %,$,presure,temp) its better looking on one chart than 5 different. It is popular, believe me, I sow more than 3 issues in this lib and I think there  are more devs, that looking for that functionality, but doesn`t ask about it here.
This lib is very good, the best chart lib on swift, I will try to develop.

**Comment by liuxuan30** [comment]
sure. I understand there are needs, but putting so many axis and lines, bars on a smart phone screen, maybe need to re consider. Any way, I don't think it's hard to achieve, but we are quite short handed :(
